orga/tests/unit/controllers/authenticated/certifications_test.js

```diff
@@ -143,4 +143,89 @@ module('Unit | Controller | authenticated/certifications', function(hooks) {
       assert.notOk(controller.isCertificationAttestationDownloadEnabled);
     });
   });
+
+  module('#downloadAttestation', function() {
+
+    test('should call the pdf file-saver service with the right parameters', async function(assert) {
```

Comment:
should call the ~~pdf~~ file-saver service with the right parameters

Ce service est aussi utilisé pour les csv